dashboard.json

```diff
@@ -46,11 +51,6 @@
                 "url": "https://pulumi.io/install/changelog.html#v141",
                 "date": "2018-06-29"
             },
```

Comment:
This trailing comma isn't valid JSON. This broke the dashboard the last time around. #483 tracks adding a test to avoid breaking this by accident.

Reply:
#497

Reply:
Thanks, Justin!